plugins/src/test/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecisionTest.java

```diff
@@ -92,6 +95,127 @@ public void preProcessShouldAddLocationIdFiltersWhenUserIsAssignedToLocationsOnl
         }
     }
 
+    @Test
+    public void
+            preProcessShouldAddAssignedRelatedEntityLocationIdsFiltersWhenUserIsAssignedToRelatedEntityLocationIdsOnly()
```

Comment:
We can rename the test case to conform to the format `<methodName><ConditionUnderTest><ExpectedBehavior>`

See https://docs.opensrp.io/engineering/android-app/developer-setup/testing/unit-tests